### PR TITLE
New post

### PR DESCRIPTION
**Pull request by Angamanga**
Hej! My second blog-post and my second pull-request ever... I hope this works, my fork is like 128 commits after the original riaguild. I have only changed the Angamanga.json file.

/Anna

### CODE REVIEW

File: src/data/members.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = {
 	jn222na: require("./users/jn222na.json"),
 	Pajn: require("./users/Pajn.json"),
 	ek222mw: require("./users/ek222mw.json"),
-	me222rs: require("./users/me222rs.json")
+	me222rs: require("./users/me222rs.json"),
+	angamanga: require("./users/angamanga.json")
```
Comment (krawaller) [inline src/data/members.js]: Here we must make `annamanga` upper-case, so the line reads like: `Angamanga: require("./users/Angamanga.json")`

File: src/data/users/angamanga.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Anna Iosif",
+  "github": "angamanga",
```
Comment (krawaller) [inline src/data/users/angamanga.json]: Same thing here, this line should be: `"github": "Angamanga",`

...and this file itself must be renamed to `Angamanga.json`!

**Comment by krawaller** [comment]
Hmm, it does NOT work, but that's probably because I renamed you from `angamanga` to `Angamanga` in order for your name to match your github name! Take a look at [my comments](https://github.com/krawaller/riaguild2015/pull/51/files) to see what you need to do! :)

**Comment by Angamanga** [comment]
Oops, I only changed the filename, i'll fix that in a minute:).

**Comment by Angamanga** [comment]
OK, provar igen med stora A!

**Comment by krawaller** [comment]
You need a big `A` at the beginning of line 22 in `members.js` too: `Angamanga: require("./users/Angamanga.json")`

...and you must also rename the `angamanga.json` file to `Angamanga.json`!

**Pull request merged**